examples/mongo_odm/demo.cpp

```diff
@@ -0,0 +1,57 @@
+// Copyright 2016 MongoDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <chrono>
+#include <string>
+#include <vector>
+
+#include <mongocxx/client.hpp>
+#include <mongocxx/instance.hpp>
+
+#include <mongo_odm/odm_collection.hpp>
+
+class Address {
+   public:
+    std::string city, state;
+    int zip_code;
+
+    template <class Archive>
+    void serialize(Archive &ar) {
+        ar(zip_code, city, state);
```

Comment:
Ah yes, I've made changes to this file since then, but I forgot that I had added it to the repo. I'll commit an updated version.